tests/index.js

```diff
@@ -50,10 +51,10 @@ describe('broccoli-gzip', function(){
 
     builder = new broccoli.Builder(tree);
     return builder.build().then(function(build) {
-      var gzippedText = fs.readFileSync(build.directory + '/test.txt.gz');
+      var gzippedText = fs.readFileSync(builder.outputPath + '/test.txt.gz');
       return RSVP.hash({
-        dir: build.directory,
-        actualCsv: fs.readFileSync(build.directory + '/test.csv'),
+        dir: builder.outputPath,
+        actualCsv: fs.readFileSync(builder.outputPath + '/test.csv'),
         actualText: RSVP.denodeify(zlib.gunzip)(gzippedText)
```

Comment:
@dfreeman since I'm not at EmberConf and sad I am coming back to this. Having some issues that I think I have pinpointed to the gunzip method. 

The tests are failing due to this error 
```{ Error: incorrect header check at Zlib._handle.onerror (zlib.js:370:17) errno: -3, code: 'Z_DATA_ERROR' }``` when you expand the `RSVP denodeify` to be a callback. Doing this because the previous error was only `incorrect header check`.

Anyways, it seems to be because of the unzip. We tried configuring the windowBits to get around it but that was not successful as well.